cdc/sinkv2/eventsink/txn/mysql/mysql_test.go

```diff
@@ -288,7 +288,8 @@ func TestNewMySQLBackendExecDML(t *testing.T) {
 	defer cancel()
 	changefeed := "test-changefeed"
 	contextutil.PutChangefeedIDInCtx(ctx, model.DefaultChangeFeedID(changefeed))
-	sinkURI, err := url.Parse("mysql://127.0.0.1:4000/?time-zone=UTC&worker-count=1")
+	sinkURI, err := url.Parse(
+		"mysql://127.0.0.1:4000/?time-zone=UTC&worker-count=1&cache-prep-stmts=false")
```

Comment:
Better to leave a TODO to test txn sink behavior when `cache-prep-stmts` is true

Reply:
the TODO is added